verification/curator-service/api/src/model/failed_attempts.ts

```diff
@@ -0,0 +1,129 @@
+import { Collection, ObjectId } from 'mongodb';
+import db from './database';
+
+const numberTimeLimiters = {
+    loginAttempt: {
+        maxNumberOfFailedLogins: 8,
+        timeWindowForFailedLogins: 60, //min
+    },
+    resetPasswordAttempt: {
+        maxNumberOfFailedLogins: 8,
+        timeWindowForFailedLogins: 60,
+    },
+    forgotPasswordAttempt: {
+        maxNumberOfFailedLogins: 8,
+        timeWindowForFailedLogins: 60,
+    },
+    resetPasswordWithTokenAttempt: {
+        maxNumberOfFailedLogins: 8,
+        timeWindowForFailedLogins: 60,
+    },
+};
+
+export enum attemptName {
```

Comment:
`enum` is treated as type declaration so to avoid confusion it's best to name it with capital letters like interfaces and types